common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -695,3 +695,11 @@ def _create_vertical_for_group(self, group, user_id):
         )
         self.children.append(dest_usage_key)  # pylint: disable=no-member
         self.group_id_to_child[unicode(group.id)] = dest_usage_key
+
+    @property
+    def tooltip_content(self):
+        if self._xmodule is None:
+            return ''
+        if self._xmodule.child is None:
+            return ''
+        return self._xmodule.child.tooltip_content if self._xmodule.child.tooltip_content else ''
```

Comment:
I think in that case, you should add a `module_attr` (https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/x_module.py#L776) for that field on the descriptor, so that the descriptor always forwards to the xmodule.